### PR TITLE
https://jira.desk.technology/browse/APIC-149

### PR DESCRIPTION
**Pull request by eldark88**
@nartiles please review

### CODE REVIEW

**Comment by nartiles** [comment]
@eldark88 :shipit:

**Comment by eldark88** [comment]
@nartiles please review the last commit

File: DeskAPIClient/DeskAPIClient/Models/DSAPIUser.h [hunk 1]
```diff
@@ -274,6 +274,22 @@
 - (NSURLSessionDataTask *)listCompanyFiltersWithParameters:(NSDictionary *)parameters
                                               queue:(NSOperationQueue *)queue
                                             success:(DSAPIPageSuccessBlock)success
-                                            failure:(DSAPIFailureBlock)failure;
+                                                   failure:(DSAPIFailureBlock)failure;
```
Comment (nartiles) [inline DeskAPIClient/DeskAPIClient/Models/DSAPIUser.h:277]: align colons

**Comment by eldark88** [comment]
@nartiles please review the last commit

**Comment by nartiles** [comment]
@eldark88 :shipit:

**Pull request merged**